bytecode/concrete.py

```diff
@@ -116,23 +116,28 @@ def disassemble(cls, lineno, code, offset):
 
 class ConcreteBytecode(_bytecode.BaseBytecode, list):
 
-    def __init__(self):
+    def __init__(self, instructions=[]):
```

Comment:
> perhaps this PR should be scrapped.

No, I didn't say that :-) I like your check method, I planned to do something similar.